src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -666,12 +671,25 @@
 
 helps['network asg'] = """
     type: group
-    short-summary: Manage application security groups.
+    short-summary: Manage application security groups (ASG).
+    long-summary: >
+        You can configure network security as a natural extension of an application's structure, ASG allows
+        you to group virtual machines and define network security policies based on those groups. You can specify an
+        application security group as the source and destination in a NSG security rule. For more information
+        visit https://docs.microsoft.com/en-us/azure/virtual-network/create-network-security-group-preview
```

Comment:
If I remember correctly, @JasonRShaver specifically asked we include URL's to the services in these descriptions.

Reply:
I'm not aware of any push to avoid URLs in the command summaries.

Reply:
Let's leave them in for now. "Further reading" URLs are a help metadata field I've wanted to discuss with engineering for a while because they make website rendering ugly and flood `--help`.